### ISSUE TITLE
reset_state/reset_states methods in F-metrics 

### ISSUE THREAD

**Issue by MayStepanyan** [tensorflow/addons#2541]
**System information**
Doesn't matter

**Describe the issue**

I'm training a model using tfa.metrics.F1Score, and after every epoch I'm getting the same message:

`/usr/local/lib/python3.6/dist-packages/tensorflow/python/keras/metrics.py:257: UserWarning: Metric F1Score implements a `reset_states()` method; rename it to `reset_state()` (without the final "s"). The name `reset_states()` has been deprecated to improve API consistency.`

The issue is [here](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/metrics/f_scores.py), line 206-210

Although I'm using callbacks and saving logs, its still very good to follow metrics/loss values during training right in the terminal. Sure, I can rename it for myself, but it's only a temporal solution. 
Please resolve this issue in your future releases

Thanks

**Comment by bhack** [comment]
It was a https://github.com/tensorflow/addons/issues/2528 dub and it was already fixed on master/nightly https://github.com/tensorflow/addons/pull/2533
